devlab/tests/utils.py

```diff
@@ -123,6 +125,32 @@ class MigrationUtils(object):
 
     def __init__(self, conf):
         self.config = conf
+        self.start_ssh_agent()
+
+    def start_ssh_agent(self):
+        if self.config.local_rsa_key_path:
```

Comment:
Might actually be better with `local_rsa_key_path` set to `~/.ssh/id_rsa` by default